docs/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ title: Changelog
 
 ## main
 
+* Fix Ruby versions and update Rails versions used in CI.
```

Comment:
```suggestion
* Correctly type Ruby version strings and update Rails versions used in CI configuration.
```

'Fix' would potentially also imply that we had locked CI to specific Ruby versions, but I suppose this would take the latest patch for each minor version.